packages/strapi-admin/admin/src/containers/ProfilePage/index.js

```diff
@@ -64,6 +75,83 @@ const ProfilePage = () => {
             })}
           </FormBloc>
         </ContainerFluid>
+
+        <ContainerFluid padding="18px 30px 18px 30px">
+          <Bloc>
+            <Padded top left right size="md">
+              <Padded top bottom size="sm">
+                <Padded top bottom size="sm">
+                  <Title>
+                    {formatMessage({ id: 'Settings.profile.form.section.password.title' })}
+                  </Title>
+                </Padded>
+              </Padded>
+
+              <Flex>
+                <div style={{ flex: 1 }}>
+                  <IntlInput
+                    label="Auth.form.password.label"
+                    type="password"
+                    autoComplete="new-password"
+                    validations={{}}
+                    error={formErrors.password}
+                    name="password"
+                    onChange={handleChange}
+                    value={get(modifiedData, 'password', '')}
+                  />
+                </div>
+
+                <Spacer />
+
+                <div style={{ flex: 1 }}>
+                  <IntlInput
+                    label="Auth.form.confirmPassword.label"
+                    type="password"
+                    validations={{}}
+                    error={formErrors.confirmPassword}
+                    name="confirmPassword"
+                    onChange={handleChange}
+                    value={get(modifiedData, 'confirmPassword', '')}
+                  />
+                </div>
+              </Flex>
+            </Padded>
+          </Bloc>
+        </ContainerFluid>
+
+        <ContainerFluid padding="18px 30px 18px 30px">
+          <Bloc>
+            <Padded top left right bottom size="md">
+              <Padded top bottom size="sm">
+                <Padded top bottom size="sm">
+                  <Title>
+                    {formatMessage({ id: 'Settings.profile.form.section.experience.title' })}
+                  </Title>
+                </Padded>
+              </Padded>
+
+              <ProfilePageLabel htmlFor="">Interface language</ProfilePageLabel>
+
+              <div style={{ width: 'calc(50% - 3rem)' }}>
+                <Select
+                  aria-labelledby="interface-language"
+                  selectedValue={currentLanguage}
+                  onChange={selectLanguage}
+                >
+                  {languages.map(language => {
+                    const langName = languageNativeNames[language];
+
+                    return (
+                      <Option value={language} key={language}>
+                        {langName}
+                      </Option>
+                    );
+                  })}
+                </Select>
```

Comment:
ℹ️  a Select component that looks like the select of HTML and works quite the same it terms of APIs